ticdc/troubleshoot-ticdc.md

```diff
@@ -109,20 +111,35 @@ If you use MySQL in a special public cloud environment, such Alibaba Cloud RDS,
     {{< copyable "shell-regular" >}}
 
     ```shell
-    cdc cli changefeed create --sink-uri="mysql://root@127.0.0.1:3306/" --tz=Asia/Shanghai
+    cdc cli changefeed create --sink-uri="mysql://root@127.0.0.1:3306/?time-zone=CST"
     ```
 
     > **Note:**
     >
-    > In MySQL, CST refers to the China Standard Time (UTC+08:00). Usually you cannot use `CST` directly in your system, but use `Asia/Shanghai` instead.
+    > CST might be an abbreviation for the following four different time zones:、
+    >
+    > + Central Standard Time (USA) UT-6:00
+    > + Central Standard Time (Australia) UT+9:30
+    > + China Standard Time UT+8:00
+    > + Cuba Standard Time UT-4:00
+    >
+    > In China, CST usually means China Standard Time, and pay attention to the discrimination when using it.
 
-Be cautious when you set the time zone of the TiCDC server, because the time zone will be used for the conversion of time type. It is recommended that you use the same time zone in the upstream and downstream databases, and specify the time zone using `--tz` when you start the TiCDC server.
+## How to understand the relationship between TiCDC time zone and upstream and downstream databases time zones?
 
-The TiCDC server chooses its time zone in the following priority:
+||Upstream time zone| TiCDC time zone|Downstream time zone|
+| :-: | :-: | :-: | :-: |
+| Configuration method | See [Time Zone Support](/configure-time-zone.md) | The `--tz` parameter when starting ticdc server | The `time-zone` parameter in `sink-uri` |
+| Description | The time zone of the upstream TiDB affects DML operations of timestamp type and DDL operations related to timestamp type columns.| TiCDC assumes that the upstream TiDB's time zone is the same as the TiCDC time zone configuration, and performs related operations on the timestamp column.| The downstream MySQL processes the timestamp contained in the DML and DDL operations according to the downstream time zone settings.|
 
-1. TiCDC first uses the time zone specified by `--tz`.
-2. When `--tz` is not available, TiCDC tries to read the time zone set by the `TZ` environment variable.
-3. When the `TZ` environment variable is not available, TiCDC uses the default time zone of the machine.
+ > **Note:**
+ >
+ > Be cautious when you set the time zone of the TiCDC server, because the time zone will be used for the conversion of time type. You should keep the upstream time zone, TiCDC time zone and downstream time zone consistent.
+ > The TiCDC server chooses its time zone in the following priority:
```

Comment:
```suggestion
 > Be careful when you set the time zone of the TiCDC server, because this time zone is used for converting the time type. Keep the upstream time zone, TiCDC time zone, and the downstream time zone consistent. The TiCDC server chooses its time zone in the following priority:
```